### PR TITLE
Travis CI: Run codespell and flake8 tests

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,10 @@ matrix:
 before_install:
     - pip install pipenv
     - pipenv install --dev --skip-lock
+    - pip install codespell flake8
```
Comment (alxwrd) [inline .travis.yml]: Could you move these dependencies into the `Pipfile` under the dev dependencies.

File: .travis.yml [hunk 2]
```diff
@@ -10,7 +10,10 @@ matrix:
...
 script:
+    - codespell -L paket --skip=./textstat/resources/*/easy_words.txt
```
Comment (alxwrd) [inline .travis.yml]: Could we preface this with `pipenv run` to make sure we're using the virtual envs `codespell`?

File: .travis.yml [hunk 3]
```diff
@@ -10,7 +10,10 @@ matrix:
...
     - pipenv run pycodestyle textstat/
     - pipenv run pycodestyle test.py
+    - flake8 .
```
Comment (alxwrd) [inline .travis.yml]: Same here, could we preface this with `pipenv run`?

File: Pipfile [hunk 4]
```diff
@@ -8,6 +8,6 @@ Pyphen = "*"
 "repoze.lru" = "*"
 
 [dev-packages]
-pycodestyle = "*"
-pytest = "*"
```
Comment (alxwrd) [inline Pipfile]: hi @cclauss, you appear to have dropped `pytest` as a dependency here. Travis will still pass as it's preinstalled there, but we will need to keep this for local tests.

File: .travis.yml (hunk 1 above)
Comment (alxwrd) [reply .travis.yml]: Don't forget to remove this please 🙂

File: Pipfile (hunk 4 above)
Comment (cclauss) [reply Pipfile]: Sorry!!!

**Comment by alxwrd** [comment]
🎉 excellent, thank you for this!

**Pull request merged**